### DISCUSSION TITLE
New Node-red update 14.5.0 gives following warning

### DISCUSSION THREAD

**Discussion by fixx-er** [zachowj/node-red-contrib-home-assistant-websocket#1059] [Q&A] [answered]
I was informed to contact author.  Is this you?

I am not sure how to handle this.

9/14/2023, 8:25:25 AM
msg : string[270]
"Deprecated API warning: Calls to RED.util.evaluateJSONataExpression must include a callback. This will not be optional in Node-RED 4.0. Please identify the node from the following stack and check for an update on npm. If none is available, please notify the node author."
9/14/2023, 8:25:25 AM
msg : string[1178]
"Error: ↵    at Object.evaluateJSONataExpression (/opt/node_modules/@node-red/util/lib/util.js:775:18)↵    at JSONataService.evaluate (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/JSONataService.js:58:39)↵    at ComparatorService.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/ComparatorService.js:47:91)↵    at CurrentStateController.onInput (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/current-state/CurrentStateController.js:48:102)↵    at CurrentStateController._InputOutputController_preOnInput (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/controllers/InputOutputController.js:61:76)↵    at /opt/node_modules/@node-red/runtime/lib/nodes/Node.js:210:26↵    at Object.trigger (/opt/node_modules/@node-red/util/lib/hooks.js:166:13)↵    at currentStateNode.Node._emitInput (/opt/node_modules/@node-red/runtime/lib/nodes/Node.js:202:11)↵    at currentStat..."
9/14/2023, 8:25:46 AMnode: sum min
status : msg.payload : number
1
9/14/2023, 8:25:47 AM
msg : string[270]
"Deprecated API warning: Calls to RED.util.evaluateJSONataExpression must include a callback. This will not be optional in Node-RED 4.0. Please identify the node from the following stack and check for an update on npm. If none is available, please notify the node author."
9/14/2023, 8:25:47 AM
msg : string[754]
"Error: ↵    at Object.evaluateJSONataExpression (/opt/node_modules/@node-red/util/lib/util.js:775:18)↵    at JSONataService.evaluate (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/JSONataService.js:58:39)↵    at ComparatorService.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/ComparatorService.js:108:94)↵    at PollState.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/BaseNode.js:128:39)↵    at PollState.onTimer (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/poll-state/controller.js:82:30)↵    at listOnTimeout (node:internal/timers:569:17)↵    at processTimers (node:internal/timers:512:7)"

Error: 
    at Object.evaluateJSONataExpression (/opt/node_modules/@node-red/util/lib/util.js:775:18)
    at JSONataService.evaluate (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/JSONataService.js:58:39)
    at ComparatorService.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/ComparatorService.js:47:91)
    at CurrentStateController.onInput (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/current-state/CurrentStateController.js:48:102)
    at CurrentStateController._InputOutputController_preOnInput (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/controllers/InputOutputController.js:61:76)
    at /opt/node_modules/@node-red/runtime/lib/nodes/Node.js:210:26
    at Object.trigger (/opt/node_modules/@node-red/util/lib/hooks.js:166:13)
    at currentStateNode.Node._emitInput (/opt/node_modules/@node-red/runtime/lib/nodes/Node.js:202:11)
    at currentStat...

**Comment by zachowj** [comment, marked as answer]
https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/898
